engine-tests/src/tests/one_inch.rs

```diff
@@ -26,7 +26,7 @@ fn test_1inch_liquidity_protocol() {
 
     let (result, profile, pool_factory) = helper.create_pool_factory(&deployer_address);
     assert!(result.gas_used >= 2_800_000); // more than 2.8M EVM gas used
-    assert_gas_bound(profile.all_gas(), 9); // less than 9 NEAR Tgas used
+    assert_gas_bound(profile.all_gas(), 10); // less than 9 NEAR Tgas used
```

Comment:
Gas costs going up a little is not ideal. But I think it is worth the trade off of not having panics in sputnikvm.

Reply:
In that case, it's `10%`. If you think it's normal - then no problem.

But also, we should charge gas for non-validation errors, from execution.